src/components/models/Gateway.ts

```diff
@@ -1,11 +1,22 @@
 import Node from "./Node";
 
 interface Gateway {
+
+  /**
+   * The unique identifier for this gateway
+   */
   uid: string;
+
+  /**
+   * The name for this gateway. 
+   */
   name: string;
   lastActivity: string;
   location?: string;
   voltage: number;
+
+  // todo add other sensor readings, e.g. temperature
```

Comment:
does a gateway need to know about sensor readings? Or just the sensor model itself?

Reply:
This is the old model - it will be phased out eventually. I'll remove the comment. The intent is that Gateways have other "sensors", like wifi/cellular signal strength. These are present in the new AppModel.